baseclasses/BaseSolver.py

```diff
@@ -28,12 +28,15 @@ def __init__(self, name, category={}, def_options={}, options={}):
 
         # Initialize Options
         for key, value in self.defaultOptions.items():
```

Comment:
I'm not super familiar with this type of Python syntax, but would `for key, (optionType, optionValue) in self.defaultOptions.items()` work? That would eliminate the following line and make it very clear what is happening.

Reply:
I tried that and it did not work. `.items()` returns a tuple with two elements.

Reply:
Oh wait, I see what you mean. I did not try it exactly like that.